packages/terra-base/src/Base.jsx

```diff
@@ -74,19 +74,16 @@ class Base extends React.Component {
   render() {
     const {
       children,
-      locale,
       customMessages,
       strictMode,
-      translationsLoadingPlaceholder,
-      ...customProps
     } = this.props;
 
     const messages = Object.assign({}, this.state.messages, customMessages);
     const renderChildren = strictMode ? (<React.StrictMode>{children}</React.StrictMode>) : children;
 
     if (!this.state.areTranslationsLoaded) return <div>{this.props.translationsLoadingPlaceholder}</div>;
```

Comment:
Probably doesn't matter either way, but alternatively this could have used the `translationsLoadingPlaceholder` from the removed props spread above and dropped `this.props`.